scripts/scil_compute_fodf_metrics.py

```diff
@@ -156,10 +156,10 @@ def main():
                                      nbr_processes=args.nbr_processes)
 
     # Computing maps
-    nufo_map, afd_max, afd_sum, rgb_map, \
-        _, _ = maps_from_sh(data, peak_dirs,
-                                       peak_values, peak_indices,
-                                       sphere, nbr_processes=args.nbr_processes)
+    if args.nufo or args.afd_max or args.afd_total or args.afd_sum or args.rgb:
+        nufo_map, afd_max, afd_sum, rgb_map, \
```

Comment:
remove import os at the begining + format parser the same way you did the others